seekers/game/game.py

```diff
@@ -38,8 +43,8 @@ def __init__(self, local_ai_locations: typing.Iterable[str], config: Config,
             self._logger.warning("Config option `global.wait-for-players=false` is not supported for local players.")
 
         if grpc_address and len(self.players) < config.global_players:
+            from seekers.grpc.server import GrpcSeekersServer
```

Comment:
This try/except statement exists to catch the import error. It existed, such that Seekers could still be used, even if the grpc stubs weren't compiled. It doesn't make sense to move the import statement outside of it.